### ISSUE TITLE
Tabs are not shown

### ISSUE THREAD

**Issue by prizrak9** [mbnuqw/sidebery#77]
my old tabs like newly created ones are not displayed in the bar.
![image](https://user-images.githubusercontent.com/26070759/64227139-efa8d880-ceea-11e9-92fc-81e981fbf038.png)

**Comment by prizrak9** [comment]
Oh, it just needed to reopen the bar. Restart of the browser had no effect tho.

**Comment by prizrak9** [comment]
After another browser restart all broke again and this time F1 won't help. That pinned tab is shown for some reason but even if I unpin it, it doesn't go anywhere(it is truly unpinned) and the option "unpin" still present.

**Comment by mbnuqw** [comment]
Looks like upgrading issue. Can you screenshot 'Storage' section in the debug page?
Thanks.

**Comment by mbnuqw** [comment]
Also you can backup addon data (Setting/Help/Export data) and try to reset settings.

**Comment by shokinn** [comment]
Hey, same Problem here.

Here is the requested screenshot.
![Screenshot 2019-09-04 at 08 46 15](https://user-images.githubusercontent.com/10045300/64231673-9348b780-cef0-11e9-840d-d1d7338f58db.png)

Resetting the addon settings works (at least for myself).
After importing it, it's breaking again. So it looks like it is some settings issue.

**Comment by mbnuqw** [comment]
@shokinn, thanks for the info and sorry for that bug.

**Comment by berdon** [comment]
![image](https://user-images.githubusercontent.com/898462/64290414-acbb2500-cf2b-11e9-9bba-8ad249574b57.png)

**Comment by berdon** [comment]
```
  if ('prev_act' === e.activateAfterClosing) {
    let i;
    for (let o = e.actTabs.length; o--; ) if (i = e.actTabs[o], (!r || !r.includes(i)) && i !== t.id && e.tabsMap[i]) {
      n = e.tabsMap[i];
      break
    }
  }
```

e.actTabs is null

**Comment by berdon** [comment]
**Workaround**
- Change "After closing current tab activate" to none
- Close Sidebery (hit the x)
- Re-open Sidebery

**Comment by mbnuqw** [comment]
Thank you all and @berdon especially.
Fixed in v3.0.3
